scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -356,7 +353,17 @@
                              ; We don't want to add in the cook-init cruft or the cook sidecar, because we
                              ; don't need them for synthetic pods and all they will do is slow things down.
                              :pod-supports-cook-init? false
-                             :pod-supports-cook-sidecar? false}))
+                             :pod-supports-cook-sidecar? false
+                             :task-id (str api/cook-synthetic-pod-name-prefix "-" pool-name "-" uuid)
+                             :task-request {:scalar-requests
+                                            (cond->> (walk/stringify-keys (tools/job-ent->resources job))
+                                                     ; We need to account for any sidecar resource requirements that the
+                                                     ; job we're launching this synthetic pod for will need to use.
+                                                     sidecar-resource-requirements
+                                                     (merge-with +
+                                                                 {"cpus" (:cpu-request sidecar-resource-requirements)
+                                                                  "mem" (:memory-request sidecar-resource-requirements)}))
+                                            :job {:job/pool {:pool/name synthetic-task-pool-name}}}}))
```

Comment:
These were moved down here just to make the map keys be lexicographically sorted.

Reply:
Thank you!